Helm quick-start guide for operator, turtles and rancher (#12)
rancher-sandbox/rancher-turtles-docs: docs/getting-started/rancher.md

# Rancher Setup

> ## Installing rancher

Reviewer: ```suggestion
## Installing Rancher
```

> In order to install rancher in existing/new k8s cluster, aside from following one of the [official installation guides](https://ranchermanager.docs.rancher.com/pages-for-subheaders/installation-and-upgrade), rancher should have `embedded-cluster-api` feature disabled. To do so, when installing rancher in the k8s cluster, the following feature flag should be specified with the `helm` command:

Reviewer: ```suggestion
In order to install Rancher in an existing/new k8s cluster, in addition to following one of the [official installation guides](https://ranchermanager.docs.rancher.com/pages-for-subheaders/installation-and-upgrade), Rancher should have `embedded-cluster-api` feature disabled. To do so, when installing Rancher in the k8s cluster, the following feature flag should be specified with the `helm` command:
```

```bash
helm install rancher rancher-stable/rancher
	--set features=embedded-cluster-api=false # Disabling embedded CAPI feature
	--set hostname=<rancher-hostname>
	--version v2.7.5
	--namespace cattle-system
	--create-namespace --wait
```

> ## Existing rancher installation

Reviewer: ```suggestion
## Existing Rancher installation
```

> To allow Rancher Turtles to be installed in the existing rancher cluster, `embedded-cluster-api` feature should be created from the manifest first:

Reviewer: ```suggestion
To allow Rancher Turtles to be installed in an existing Rancher cluster, the `embedded-cluster-api` feature flag can be disabled by applying the following manifest:
```
